.dependabot/config.yml

```diff
@@ -110,7 +110,7 @@ update_configs:
       - match:
           dependency_name: "org.testcontainers:testcontainers"
       - match:
-          dependency_name: "org.testcontainers:junit-jupiter"
+          dependency_name: "org.testcontainers:postgresql"
```

Comment:
@gastaldi when you're back and you unpile your emails, I have a question for you: does dependabot support wildcards? I couldn't find any information about it. It could be handy for testcontainers to have a wildcard for all the group id for instance.

Reply:
That would be nice